saveload.py

```diff
@@ -1,117 +1,99 @@
-import config
-import jax.numpy as np
 import os
+import logging
+
+import jax.numpy as np
+
+from config import config
 
-dirname = os.path.dirname(__file__)
-pathed = os.path.join(dirname, "saves\save")
+# dirname = os.path.dirname(__file__)
```

Comment:
This is a feature that I am not sure if we want and that was also not implemented correctly. I believe they were trying to save the checkpoint files in a `saves` subdirectory of the same directory as the program. I changed it to save the checkpoint files directly in the current working directory, which is what Prime95/MPrime, Mlucas and CUDALucas do, although I also added a `-w/--dir/--workdir` option so this could be changed. GpuOwl saves them in a subdirectory for each exponent.